components/cards/Cards.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from "./Cards.module.css";
+
+export type Props = {
+  cardrecipesrc: string;
+  cardrecipename: string;
+  cardopenrecipesrc: string;
+};
+
+function RecipeCard({
+  cardrecipesrc,
+  cardrecipename,
+  cardopenrecipesrc,
```

Comment:
This naming would be clearer, also removed the prop for the button imgSrc as this doesn't need to change this often.
```suggestion
  imgSrc,
  name,
```